ocsp_status_test.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"crypto"
+	"io/ioutil"
+	"log"
+	"os"
+	"testing"
+)
+
+const (
+	GoodResp = "./testdata/resps/oldfbresp"
+	GoodCert = "./testdata/certs/google.der"
+	ExpiredCert = "./testdata/certs/expiredcert.der"
+	NoIssuingURLCert = "./testdata/certs/rootcert.der"
+	GoodURL = "google.com:443"
+	RevokedURL = "revoked.grc.com:443"
```

Comment:
Is there a way to mock these responses so that they won't be dependent on outside sources functioning in an assumed way?